test/bdd/features/gatekeeper_api.feature

```diff
@@ -93,3 +93,45 @@ Feature: Gatekeeper API
     When  an HTTP GET with "(request-target),date" headers signed by "Handler" is sent to "https://localhost:9014/v1/release/{ticket_id}/status"
     Then  response status is "200 OK"
      And  response contains "status" with value "READY_TO_COLLECT"
+
+    Scenario: Collect and extract data from DID
+      Given did owner with name "Intake Processor"
+      And did owner with name "Handler"
+      And did owner with name "Approver 1"
+      And did owner with name "Approver 2"
+      And policy configuration with ID "authorize-policy"
+          """
+          {
+            "collectors": ["{{ .GetDID "Intake Processor" }}"],
+            "handlers": ["{{ .GetDID "Handler" }}"],
+            "approvers": ["{{ .GetDID "Approver 1" }}", "{{ .GetDID "Approver 2" }}"],
+            "min_approvers": 2
+          }
+          """
+      And social media handle "@thanos27" converted into DID by "Intake Processor"
+      And release transaction created on DID by "Handler"
+
+      When  an HTTP POST with "(request-target),date" headers signed by "Approver 1" is sent to "https://localhost:9014/v1/release/{ticket_id}/authorize"
+      Then  response status is "200 OK"
+
+      When  an HTTP GET with "(request-target),date" headers signed by "Handler" is sent to "https://localhost:9014/v1/release/{ticket_id}/status"
+      Then  response status is "200 OK"
+      And  response contains "status" with value "COLLECTING"
+
+      When  an HTTP POST with "(request-target),date" headers signed by "Approver 2" is sent to "https://localhost:9014/v1/release/{ticket_id}/authorize"
+      Then  response status is "200 OK"
+
+      When  an HTTP GET with "(request-target),date" headers signed by "Handler" is sent to "https://localhost:9014/v1/release/{ticket_id}/status"
+      Then  response status is "200 OK"
+      And  response contains "status" with value "READY_TO_COLLECT"
```

Comment:
I would suggest encapsulating the flow of getting all approvals into a single statement under `Given` section. E.g.
```
Given ...
   And release transaction created on DID by "Handler"
   And ticket got approved by "Approver 1, Approver 2"
...
```